### PR TITLE
Replacing text for boundary among types of ruby, and add dfn (B-#46-3)

### LINKED ISSUE #61: Replace the last sentence of section 3.1 (B-#46-3)
from https://lists.w3.org/Archives/Public/public-i18n-japanese/2021OctDec/0104.html

New Japanese text proposed, so replace the last sentence of section 3.1 with a new text.

### PR DESCRIPTION
**Pull request by himorin**
closes #61

For second commit (to add dfn), added as an example and I would revert it if not applicable. (main motivation is to refer definition of types from text in section 3.x, e.g. first paragraph in mono-ruby)

***
<a href="https://pr-preview.s3.amazonaws.com/himorin/simple-ruby/pull/63.html" title="Last updated on Nov 29, 2021, 6:26 AM UTC (c07bae6)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/simple-ruby/63/8c0efc9...himorin:c07bae6.html" title="Last updated on Nov 29, 2021, 6:26 AM UTC (c07bae6)">Diff</a>

***
<a href="https://pr-preview.s3.amazonaws.com/himorin/simple-ruby/pull/63.html" title="Last updated on May 12, 2023, 7:51 AM UTC (6b45ebd)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/simple-ruby/63/6a303c9...himorin:6b45ebd.html" title="Last updated on May 12, 2023, 7:51 AM UTC (6b45ebd)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -266,11 +266,23 @@ <h3 id="types-of-ruby">Types of ruby</h3>
...
 <p>Which one to use depends on the relationship
 between the ruby and the base characters.
-Mono-ruby is used to connect ruby to a single base character,
-Jukugo-ruby is used when multiple base characters each have a corresponding ruby
+<a title="mono-ruby">Mono-ruby</a> is used to connect ruby to a single base character,
+<dfn>Jukugo-ruby</dfn> is used when multiple base characters each have a corresponding ruby
 and at the same time the whole group needs to be processed together,
-and group-ruby is used when ruby is attached to a group of base characters together (see <a href="#ruby-types"></a>).
-Each is used when specified.</p>
+and <dfn>group-ruby</dfn> is used when ruby is attached to a group of base characters together (see <a href="#ruby-types"></a> for examples).
```
Comment (xfq) [inline index.html]:
```suggestion
and <dfn>Group-ruby</dfn> is used when ruby is attached to a group of base characters together (see <a href="#ruby-types"></a> for examples).
```

It seems that the first letters of mono/jukugo/group-ruby in this paragraph are all capitalized. If we decide this way, this one should also be capitalized.

Comment (himorin) [reply index.html]: Thank you for catching! (I also wonder we should use capitalized for whole document...)

File: index.html [hunk 2]
```diff
@@ -266,11 +266,23 @@ <h3 id="types-of-ruby">Types of ruby</h3>
...
 between the ruby and the base characters.
-Mono-ruby is used to connect ruby to a single base character,
-Jukugo-ruby is used when multiple base characters each have a corresponding ruby
+<a title="mono-ruby">Mono-ruby</a> is used to connect ruby to a single base character,
+<dfn>Jukugo-ruby</dfn> is used when multiple base characters each have a corresponding ruby
 and at the same time the whole group needs to be processed together,
-and group-ruby is used when ruby is attached to a group of base characters together (see <a href="#ruby-types"></a>).
-Each is used when specified.</p>
+and <dfn>Group-ruby</dfn> is used when ruby is attached to a group of base characters together (see <a href="#ruby-types"></a> for examples).
+<dfn>Mono-ruby</dfn> is defined to have a single character as its ruby base, it is clear to tell as <a title="mono-ruby">Mono-ruby</a>
```
Comment (himorin) [inline index.html]: compared with v22/23, new paragraph need to be added here.

File: index.html [hunk 3]
```diff
@@ -266,11 +266,23 @@ <h3 id="types-of-ruby">Types of ruby</h3>
...
-and group-ruby is used when ruby is attached to a group of base characters together (see <a href="#ruby-types"></a>).
-Each is used when specified.</p>
+and <dfn>Group-ruby</dfn> is used when ruby is attached to a group of base characters together (see <a href="#ruby-types"></a> for examples).
+<dfn>Mono-ruby</dfn> is defined to have a single character as its ruby base, it is clear to tell as <a title="mono-ruby">Mono-ruby</a> 
+when <a title="ruby block">ruby block</a> has a single character as its ruby base. 
+But, there is no clear boundary on handling between <a title="group-ruby">Group-ruby</a> and <a title="jukugo-ruby">Jukugo-ruby</a>, both of whose ruby base contains multiple characters. 
+In this document, 
+a <a title="ruby block">ruby block</a> with only one group of ruby annotation and ruby base consisted of multiple characters is handled as <a title="group-ruby">Group-ruby</a>, 
+and a <a title="ruby block">ruby block</a>, 
+whose ruby base is consisted of multiple characters and individual ruby annotation is assigned to each characrets as its ruby base,
```
Comment (himorin) [inline index.html]:
```suggestion
whose ruby base is consisted of multiple characters and individual ruby annotation is assigned to each character as its ruby base, 
```

**Review by kidayasuo — APPROVED** [review]
The changes look good. Reviewed with Shimono-san.

**Comment by himorin** [comment]
conflict resolved.
text added by bef62e8 was disappeared by unknown reason during git rebase, but recovered by 6b45ebd.

**Pull request merged**